testing/go/set_test.go

```diff
@@ -82,13 +82,21 @@ var setStmts = []ScriptTest{
 				Expected: []sql.Row{{"\"$user\", public"}},
 			},
 			{
-				Query:    "SET SCHEMA 'postgres';",
+				Query:    "SET SCHEMA postgres;",
```

Comment:
I just tried this locally with
```sql
SET SCHEMA 'postgres';
```
and
```sql
SET SCHEMA postgres;
```
And it only works with the quotation marks. Seems like the new form is incorrect from my testing. Why did we change it?

Reply:
Ah I missed this case. The syntax `SET search_path = 'postgres'` and SET search_path = postgres` works the same, which is why I made this change. I'll make the fix for SET special cases like `SET SCHEMA ...`.

Reply:
I see, that makes sense!